packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js

```diff
@@ -0,0 +1,270 @@
+        expect($eyebrow).not.to.be.empty;
+      });
+    });
+
+    it('should render heading content', () => {
+      cy.get(_selectors.heading).then($heading => {
+        expect($heading).not.to.be.empty;
+      });
+    });
+
+    it('should render body copy content', () => {
+      cy.get(_selectors.copy).then($copy => {
+        expect($copy).not.to.be.empty;
+      });
+    });
+
+    it('should render footer content', () => {
+      cy.get(_selectors.footer).then($footer => {
+        expect($footer).not.to.be.empty;
+      });
+    });
+  },
+  checkClickableCard: () => {
+    it('should check for link', () => {
+      cy.get(`${ddsPrefix}-card > ${ddsPrefix}-card-footer`)
+        .shadow()
+        .find(`a.${prefix}--card__footer`)
+        .then($link => {
+          const url = $link.prop('href');
+          expect(url).not.to.be.empty;
+        });
+    });
+
+    it("should check that the footer's pseudo class takes up entire card to be clickable", () => {
+      cy.get(`${ddsPrefix}-card > ${ddsPrefix}-card-footer`)
+        .shadow()
+        .find('a')
+        .then($els => {
+          const win = $els[0].ownerDocument.defaultView;
+          const after = win.getComputedStyle($els[0], ':after');
+          const positionValue = after.getPropertyValue('position');
+          const insetValue = after.getPropertyValue('inset');
+
+          expect(positionValue).to.eq('absolute');
+          expect(insetValue).to.eq('0px');
+        });
+    });
+  },
+  checkNonClickableCard: () => {
+    it('should not respond to a click ', () => {
+      let initialLocation;
+      cy.location('href')
+        .then(location => {
+          initialLocation = location;
+        })
+        .get(_selectorBase)
+        .click()
+        .location('href')
+        .then(location => {
+          expect(location).to.equal(initialLocation);
+        });
+    });
+  },
+  checkImageRenders: () => {
+    cy.get(_selectors.image).should('have.length', 1);
+    cy.takeSnapshots();
+  },
+  checkTagGroupRenders: () => {
+    cy.get(_selectors.tagGroup).should('have.length', 1);
+    cy.takeSnapshots();
+  },
+  checkOutlineRenders: () => {
+    cy.get(_selectorBase).should('have.attr', 'border');
+    // converted HEX var(--cds-ui-03, #e0e0e0) to RGB
+    cy.get(_selectorBase)
+      .should('have.css', 'border')
+      .and('equal', '1px solid rgb(224, 224, 224)');
+
+    cy.get(_selectorBase).should('have.attr', 'color-scheme', 'light');
+    // converted HEX var(--cds-ui-02, #ffffff) to RGB
+    cy.get(_selectorBase)
+      .shadow()
+      .find(`.${prefix}--card__wrapper`)
+      .should('have.css', 'background-color')
+      .and('equal', 'rgb(255, 255, 255)');
+
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
+    // cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+    //  widths: [1280],
+    // });
```

Comment:
```suggestion
    cy.takeSnapshots();
```

I think you just missed these?